tests/filters/test_value_array_dictionary_check.py

```diff
@@ -11,11 +11,11 @@ class TestValueArrayDictionaryCheck:
     @pytest.fixture
     def token_rule(self, config) -> Rule:
         token_rule_without_filters = {
-            "name": "Token",
+            "name": "Pass",
             "severity": "medium",
             "type": "keyword",
-            "values": ["token"],
-            "filter_type": [VariableNotAllowedPatternCheck.__name__],
+            "values": ["pass"],
+            "filter_type": [ValueArrayDictionaryCheck.__name__],
             "use_ml": True,
             "min_line_len": 0,
             "validations": [],
```

Comment:
Why you changed the `Token` to `Pass`?
Isn't `token_rule()` method testing about `Token`?

Reply:
I see thank you for explanation.